frontend/src/searchContext.tsx

```diff
@@ -96,18 +105,38 @@ type Store = {
 const SearchContext = createContext<Store | undefined>(undefined);
 SearchContext.displayName = 'SearchContext';
 
+// Calculate upcoming season
+const dt = DateTime.now().setZone('America/New_York');
```

Comment:
Should we also handle summer courses here?

Reply:
i thought about that but assumed most people don't do summer session. think we should?